projects/storefrontlib/src/cms-components/navigation/footer-navigation/scroll-to-top/scroll-to-top.component.scss

```diff
@@ -0,0 +1,34 @@
+.cx-main {
+  position: fixed;
+  bottom: 15px;
+  right: 15px;
+  opacity: 0;
+  transition: all 0.2s ease-in-out;
+}
+.cx-show-scrollTop {
+  opacity: 1;
+  transition: all 0.2s ease-in-out;
+}
+.cx-button-scroll-to-top {
+  border: none;
+  background: none;
+  padding: 0;
+}
+.cx-img-scroll-to-top {
+  display: block;
+  content: url('../../../../assets/scroll-to-top.png');
```

Comment:
You should use our icons, not an image

Reply:
Bill had asked me to get an image from Suleem from UX design. So I got 2 images image from Suleem. If this is not ok, then can you point me which image to use?